### ISSUE TITLE
New messaging decoder

### ISSUE THREAD

**Issue by curly60e** [fusion44/sendmany#12]
I had found that if you send messages from SendMany and you want to check it for example on Juggernaut you cannot see it. I believe it's the messaging decoder. I reported it on Juggernaut's github. 
The problem is that there isn't a standard decoder for lightning.

Someone told me that there is a proposal for a Lightning Universal Messaging Objects on github.

https://github.com/stakwork/lumo-spec 

This allow to improve the capabilities of messaging for Lightning.
If you can check it out.

**Comment by fusion44** [comment]
Yes, thats on my to research list. I have a few reservations because the spec seems to require a sphinx relay server to be running alongside the LND node. Users should not need to run another server in addition to LND.
